.github/workflows/build.yml

```diff
@@ -61,26 +112,54 @@ jobs:
         with:
           import-github-users: atmoz
 
-      - name: Push images to Docker Hub registry
+      # Build & push docker image multi-platform
+      # note: the linux/amd64 image is not rebuilt, the cache is still present from the previous steps.
+      - name: Build & Push debian image to dockerhub
         if: github.ref == 'refs/heads/master'
-        run: |
-          echo "${{ secrets.DOCKER_HUB_PASSWORD }}" | docker login \
-            -u ${{ secrets.DOCKER_HUB_USERNAME }} --password-stdin
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
```

Comment:
this can be specified as a newline separated list, which is a bit more readable.
```yml
platforms: |
  linux/386
  linux/amd64
  linux/arm/v6
  linux/arm/v7
  linux/arm64
  linux/ppc64le
  linux/s390x
```